tests/tests_fabric/test_fabric.py

```diff
@@ -643,11 +643,10 @@ def test_no_backward_sync():
 
 def test_launch_without_function():
     """Test the various ways `Fabric.launch()` can be called."""
-
     # default: no launcher, single process
     fabric = Fabric()
-    with mock.patch("lightning.fabric.fabric._do_nothing") as nothing:
```

Comment:
I couldn't get patching to work because the `.launch()` method default value doesn't get replaced.